wagtail_review/models.py

```diff
@@ -423,7 +423,7 @@ def user_can_unlock(self, page, user):
 
     def get_actions(self, page, user, reviewer=None, **kwargs):
         if not reviewer:
-            reviewer = Reviewer.objects.get(internal__pk=user.pk)
+            reviewer = Reviewer.objects.get_or_create(internal=user)[0]
```

Comment:
Just a stylistic quibble... can we make this
```suggestion
            reviewer, _ = Reviewer.objects.get_or_create(internal=user)
```
instead? I think that's the more common idiom for get_or_create - the `[0]` had me confused for a moment, thinking "surely that doesn't return a list...?"

Reply:
Sure, yep! That makes sense, sorry for the confusion.